modules/installation-aws-user-infra-requirements.adoc

```diff
@@ -555,6 +555,7 @@ You need `AWS::EC2::Instance` objects for the following machines:
 
 * A bootstrap machine. This machine is required during installation, but you can remove it after your cluster deploys.
 * Three control plane machines. The control plane machines are not governed by a compute machine set.
+//todo: assess for CPMS impact. 4.11: https://docs.openshift.com/container-platform/4.11/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-cluster-machines_installing-aws-user-infra
```

Comment:
Installing > Installing on AWS > Installing a cluster on user-provisioned infrastructure in AWS by using CloudFormation templates > Required AWS infrastructure components > [Cluster machines](https://53397--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-cluster-machines_installing-aws-user-infra)

Reply:
Pretty simple fix, just say they are governed by a CPMS for platforms that support it.